index.js

```diff
@@ -9,6 +9,20 @@ var camelcaseKeys = require('camelcase-keys');
 delete require.cache[__filename];
 var parentDir = path.dirname(module.parent.filename);
 
+function handleError(err) {
+	if (!err) {
+		return;
+	}
+
+	console.error(err.message);
```

Comment:
@kevva I'm not convinced we should be doing this. It makes it really hard to debug as it hides the stack trace. Maybe throw if the `DEBUG` environment variable is set?